fastlane/Fastfile

```diff
@@ -854,36 +844,82 @@ desc 'Show current frameworks size'
 lane :show_frameworks_sizes do |options|
   next unless is_check_required(sources: sources_matrix[:size], force_check: @force_check)
 
-  options[:sizes] ||= frameworks_sizes
+  ['performance/'].each { |dir| FileUtils.remove_dir(dir, force: true) }
 
-  if is_ci && !ENV['GITHUB_PR_NUM'].to_s.empty?
-    markdown_table = "## SDK Size\n| `StreamChat` | `StreamChatUI` | `Total` |\n| - | - | - |\n"
-    markdown_table << "|#{options[:sizes][:stream_chat]}MB|#{options[:sizes][:stream_chat_ui]}MB|#{options[:sizes][:total]}MB|"
-    sh("gh pr comment #{ENV.fetch('GITHUB_PR_NUM')} -b '#{markdown_table}'")
+  sh("git clone #{performance_git} #{File.dirname(sdk_size_path)}")
+  performance_benchmarks = JSON.parse(File.read(sdk_size_path))
+  expected_performance = performance_benchmarks['develop']
+  actual_performance = options[:sizes] || frameworks_sizes
```

Comment:
At least the last 2 properties should be renamed to something more clear 🤔 
```suggestion
 // What this one is about?
  performance_benchmarks = JSON.parse(File.read(sdk_size_path))
  current_size = performance_benchmarks['develop']
  new_size = options[:sizes] || frameworks_sizes
```

Reply:
Updated 👍